src/plugins/CoreLoadPlugin.ts

```diff
@@ -330,9 +330,9 @@ export default class DojoLoadPlugin {
 				}
 			});
 
-			compilation.moduleTemplate.plugin('module', (source, module: NormalModule) => {
+			compilation.moduleTemplate.plugin('module', (source, module) => {
```

Comment:
How come removing this type and then casting?

Reply:
Strict function typing doesn't allow typing `module` as `NormalModule` in the function declaration